### PR TITLE
Updated Restaurant and Menu classes. Added getter in Dish class.

### LINKED ISSUE #2: [Feature 2] add/edit menu
Allows a Restaurant Owner to add a new menu or edit a pre-existing menu on their restaurant.

### PR DESCRIPTION
**Pull request by emjennett**

### CODE REVIEW

**Review by LJCar — APPROVED** [review]
looks good

File: src/main/java/Entities/Menu.java [hunk 1]
```diff
@@ -1,4 +1,33 @@
...
+        return title;
+    }
+
+    public String getDescription(){
+        return description;
+    }
+
+    public void storeDishes(Dish dish){
+        //Stores dishes in a hashmap. This can be used in cases of dish searching from the user.
+        HashMap<String, Dish> dishMap = new HashMap<String, Dish>();
```
Comment (Moe-Reda) [inline src/main/java/Entities/Menu.java:29]: Wouldn't it be better if the _dishMap_ was an instance variable ?

File: src/main/java/Entities/Restaurant.java [hunk 2]
```diff
@@ -1,4 +1,41 @@
 package Entities;
 
+import java.util.ArrayList;
+import java.util.HashMap;
+
 public class Restaurant {
+    private String resCategory; //Category of the restaurant
+    private String location; //Location of the restaurant
+    private Double rating; //Average rating of the restaurant from its corresponding dish ratings
+    private ArrayList<Dish> dishRatings = new ArrayList<Dish>();
```
Comment (Moe-Reda) [inline src/main/java/Entities/Restaurant.java:10]: This instance variable could be redundant sine you can already access dish ratings through the menu and dish class

**Review by Moe-Reda — COMMENTED** [review]
Looks good overall. Check the comments that I left in the _Restaurant_ and _Menu_ class.

**Comment by INCREDIJOSH** [comment]
I approve

**Review by INCREDIJOSH — APPROVED** [review]
I approve. For real this time.

**Pull request merged**